### ISSUE TITLE
Azure AI Studio

### ISSUE THREAD

**Issue by andreas-sommer-engenious** [Azure-Samples/azure-search-openai-demo#1831]
Is this sample the best way to chat with your own data or should you use Azure AI Studio in the future? What is the difference between the two technologies?

**Comment by IainD925** [comment]
Good question @andreas-sommer-engenious.  To my mind, this is the "best" way to chat with your own data.
Others recommend Azure AI Studio for its ease (it is excellent) but when you want to change the front end, you need to clone the repo for the solutions locally, make changes and then deploy that repo, as you do here.

Doesn't appear as simple then, unless someone can advise I'm missing something?

In summary - AI Studio deployments for ease and this repo for a more complete solution.

**Comment by pamelafox** [comment]
We have a comparison with the "On Your Data" sample here:
https://github.com/Azure-Samples/azure-search-openai-demo/blob/main/docs/other_samples.md
That's not Azure AI Studio though, exactly, that's Azure OpenAI Studio.

I personally prefer this approach for maximal code flexibility, but some audiences may prefer the AI Studio approach, especially if you're used to Azure ML workspaces. 

We'll have sessions about both approaches at the upcoming RAGHack:
https://github.com/microsoft/RAG_Hack
